website/screens/components/flex/code/FlexCodePage.tsx

```diff
@@ -0,0 +1,167 @@
+import { DxcFlex, DxcTable } from "@dxc-technology/halstack-react";
+import Code from "@/common/Code";
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import Example from "@/common/example/Example";
+import directionAlignment from "./examples/directionAlignment";
+import gapOrderGrow from "./examples/gapOrderGrow";
+import HeaderDescriptionCell from "@/common/HeaderDescriptionCell";
+
+const sections = [
+  {
+    title: "Props",
```

Comment:
Maybe we can add a link to Flexbox documentation like [this one ](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox).

Reply:
Also agreed! I talk this with @jsardid but i completely forgot about it.